src/Symfony/Component/Config/Tests/ConfigCacheTest.php

```diff
@@ -13,21 +13,25 @@
 
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Config\ConfigCache;
+use Symfony\Component\Config\Resource\FileResource;
 use Symfony\Component\Config\Resource\SelfCheckingResourceChecker;
 use Symfony\Component\Config\Tests\Resource\ResourceStub;
 
 class ConfigCacheTest extends TestCase
 {
     private string $cacheFile;
 
```

Comment:
```suggestion
```

Reply:
Agree to remove this one also then ?

https://github.com/symfony/symfony/blob/87186b2c6956d9985c97a1004012ba477a4f72ea/src/Symfony/Component/Config/Tests/ResourceCheckerConfigCacheTest.php#L23